CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Upcoming
 
+## 1.0.3
```

Comment:
please remove the tag from here, It should be always as upcoming until it is released